src/Entity/Contact.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Ramsey\Uuid\Uuid;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity()
+ */
+class Contact
+{
+    /**
+     * @ORM\Id()
+     * @ORM\Column(type="guid")
+     */
+    private $id;
+
+    /**
+     * @Assert\NotBlank()
+     * @ORM\Column(type="string", length=63)
+     */
+    private $firstName;
+
+    /**
+     * @ORM\Column(type="string", length=63, nullable=true)
+     */
+    private $lastName;
+
+    /**
+     * @Assert\Email()
+     * @ORM\Column(type="string", length=127, nullable=true)
+     */
+    private $email;
+
+    /**
+     * @ORM\Column(type="string", length=31, nullable=true)
+     */
+    private $phoneNumber;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="App\Entity\Address", cascade={"persist", "remove"})
```

Comment:
One contact can have many addresses? What is the use case to deal with multiple addresses ? And if so, each addresses should have a unique name like "Home", "Office", ect

Reply:
This is the need expressed in the issue

Reply:
Requirements are not set in stone, this is not a waterfall driven project management, and PR are not only a good place for technical discussion but rather for all discussion types.

Reply:
I added addresses name so that we can add something like "headquarter".

For the question one or many addresses by contact, I'm not sure we can answer now, so the simplest solution is to allow many addresses, then change it later depending on the need